src/components/ManageCentersModal.tsx

```diff
@@ -152,12 +166,22 @@ const ManageCentersModal: React.FC<ManageUsersModalProps> = ({
                         {name}
                       </Box>
                       <Box>
-                        <Checkbox
-                          sx={{ pb: '20px' }}
-                          className="checkBox_svg"
-                          checked={checkedCenters.includes(name)}
-                          onChange={() => handleToggle(name)}
-                        />
+                        {isForLearner ? (
+                          <Radio
+                            sx={{ pb: '20px' }}
+                            className="radio_svg"
```

Comment:
@suvarnakale please rename the class. It should not be a snake case

Reply:
sure.